content/competitions/hackathon-2021.html

```diff
@@ -0,0 +1,121 @@
+---
+title: "Hackathon"
+draft: false
+---
+
+<div class="section columns is-centered">
+    <section class="column is-6">
+        
+        <div class="block">
+            <a href="https://www.facebook.com/events/200547242027862/">
+                <img src="{{< baseurl >}}/img/media/competitions/hackathon_2021_banner.png" style="width: 100%" />
+            </a>
+        </div>
+
+        <h1 class="title is-3">UQCS Hackathon 2021</h1>
+        <p class="subtitle is-4">Friday 20th August &ndash; Sunday 22th August</p>
+        <p class="subtitle is-6">Registrations open at 5pm.</p>
+
+        <div class="content">
+            <div class="buttons has-text-centered">
+                <a href="https://events.humanitix.com/hackathon2021" class="button is-link uqcs-blue">Get your tickets now!</a>
+                <a href="https://www.facebook.com/events/200547242027862" class="button">Facebook event</a>
+            </div>
+
+			<p>
+				The much awaited UQCS Hackathon is back & better than ever! Taking place from the 20th to 22nd of August, this is a great time to learn a new framework, develop your programming and/or design skills, and win some money. Use the link above to buy tickets - make sure to buy them before 6th August to get a LIMITED EDITION Hackathon 2021 t-shirt for ONLY $5 and early bird tickets for only $10!
+			</p>
+
+			<p>
+				If you aren’t familiar with how the UQCS Hackathon works, it’s pretty simple:
+			</p>
+			<ol><li>Form your team 👭👬</li><li>Build something cool 💻 <br></li><li>Eat free food 🍕 </li><li>Present what you have made 🎤 <br></li><li>Win prizes 🏆 <br></li></ol>
+
+			<p>
+				In the past people have built websites, desktop apps, games, hardware, and even robots and if you’ve never been to a Hackathon before, don’t worry as we’ll be running a talk beforehand walking through what to do at your first Hackathon. Make sure to hit going on Facebook and to buy your tickets now!
+			</p>
+
+            <h3>Projects</h3>
+            <p>
+                Some past projects are on our showcase page.
+                Still not sure what to do? Check out the ideas at the link below. 
+            </p>
+            <p>
+                <a class="button" href="/showcase/">UQCS Showcase</a>
+                <a class="button" href="https://docs.google.com/document/d/1IWN7TsKN0psV9NZ7ss2g7-1qDeCXd58WMzZvUGcIsu0/edit#">Hackathon Project Ideas</a>
```

Comment:
Not sure if this might need changing? This is last year's doc with the old logo, dates, and projects that may have already been done.

Reply:
We should create a new doc with new ideas. In the meantime, just remove the link and change the text above as well.